photo/src/main/java/com/paulacr/breakfastplaces/photo/Picture.kt

```diff
@@ -24,4 +64,101 @@ fun Picture() {
     ) {
         root.controller = cameraController
     }
+
+    val fileOptions = ImageCapture.OutputFileOptions.Builder(
+        LocalContext.current.contentResolver,
+        MediaStore.Images.Media.EXTERNAL_CONTENT_URI,
+        ContentValues()
+    )
+        .build()
+
+    val callback = object : ImageCapture.OnImageSavedCallback {
+        override fun onImageSaved(outputFileResults: ImageCapture.OutputFileResults) {
+            Timber.d("qwerty saved!")
+            stateSetter(CameraState.RESULT)
+            imageUriSetter(outputFileResults.savedUri)
+        }
+
+        override fun onError(exception: ImageCaptureException) {
+            Timber.d("qwerty got error $exception")
+        }
+
+    }
+
+    val newSingleThreadExecutor = remember { Executors.newSingleThreadExecutor() }
+    FloatingActionButton(
+        onClick = {
+            cameraController.takePicture(
+                /* outputFileOptions = */ fileOptions,
```

Comment:
I don't think these comments make sense since the names are already saying what is here